### DISCUSSION TITLE
Move DB to another server with existing prisma migrations

### DISCUSSION THREAD

**Discussion by rohit12sh** [prisma/prisma#21088] [Q&A]
### Question

I have a project that is already apply DB changes using prisma migrate. I have a need to relocate DB to a new server. DB is postgres. 

How to make this move while keeping prisma migration happy with this. 

In the past, I had prisma complaining of drift amd forced me to reset data for development server. This is a No-No in my case. 

is there a clean way to do the DB relocation?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
If you only change the host where the database is run, and keep the actual data (especially in the `_prisma_migrations` table) and the Prisma migration history, identical - then there will be no drift. Drift is only detected when the local migration history, and the remote database schena do not match any more because one of the both was modified outside of `prisma migrate`.

**Reply by ludralph** [reply]
Hi @rohit12sh 👋 

Did you get a chance to see Jan's recommendation? If you have any outstanding questions, please let us know; we'd be happy to help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
